test_runner/src/main/kotlin/ftl/cli/firebase/test/android/AndroidRunCommand.kt

```diff
@@ -51,4 +51,11 @@ class AndroidRunCommand : Runnable {
           See https://developer.android.com/reference/android/support/test/runner/AndroidJUnitRunner for more
           information."""])
     var testTargets: List<String>? = null
+
+    @Option(names = ["--use-orchestrator"], description = ["""Whether each test runs in its own Instrumentation instance
+        |with the Android Test Orchestrator (default: Orchestrator is not used, same as specifying --no-use-orchestrator).
```

Comment:
> default: Orchestrator is not used

The default in flank is to use orchestrator. I think we may want to add `--no-use-orchestrator` to this PR